rc/bazel.kak

```diff
@@ -0,0 +1,35 @@
+# https://docs.bazel.build/versions/master/skylark/build-style.html
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾ 
+
+# Detection
+# ‾‾‾‾‾‾‾‾‾
+
+hook global BufCreate .+BUILD %{
+    set-option buffer filetype bazel_build
+}
+
+# Initialization
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+
+hook -group bazel-highlight global WinSetOption filetype=bazel_build %{
+        require-module bazel
+
+        add-highlighter window/bazel_build ref bazel_build
+        hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/bazel_build }
```

Comment:
The indentation here is different from in the detection hook, for some reason.

Reply:
Thanks for pointing this out, fixed.